release/sdk.yaml

```diff
@@ -21,18 +21,18 @@ groups:
     group-id: com.digitalasset
     version: __VERSION__
     packages:
-      navigator:
+      navigator_2.12:
```

Comment:
Is there an _easy_ way to drop the `2.12` again? If so, let's do it in a followup PR, if not, let's not bother.

Reply:
Sure we could certainly add the option to not add the prefix. But given that the fact that we didn’t add it to the sandbox before was a bug according to @dajmaki, I expect that there is probably a good reason for adding it to all scala targets.

Reply:
If it was a bug for sandbox, then let's keep the suffix everywhere. Thanks.